app/views/interactions/index.html.erb

```diff
@@ -43,7 +43,13 @@
           </p>
         </td>
         <td class='text-center'>
-          <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s") %>
+          <div class='actions add-vertical-margins col-xs-6 no-gutter'
+               data-module="track-click"
+               data-track-action="editButtonClick"
+               data-track-label="<%= edit_link_path(interaction_slug: interaction.slug) %>"
+               data-track-category="userInteraction:LLM">
+               <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s js-track") %>
```

Comment:
I'm not sure this will track as you intended. The module pulls the label and value from the link, and the action and category from the container. This will use the text of the link as the label, rather than the one provided in the attribute.

See: https://github.com/alphagov/govuk_admin_template/pull/145/files#diff-3018827fa99e362685623cba2c52c238

What are the styles on the wrapping `div` necessary for? If the layout has changed, could you include screenshots?